src/main/java/com/smartsheet/api/models/SheetTemplateInclusion.java

```diff
@@ -0,0 +1,43 @@
+package com.smartsheet.api.models;
+
+/*
+ * #[license]
+ * Smartsheet SDK for Java
+ * %%
+ * Copyright (C) 2014 Smartsheet
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * %[license]
+ */
+
+
+/**
+ * Represents specific objects that can be included in some responses.
+ */
+public enum SheetTemplateInclusion {
+    DATA            ("data"),
+    ATTACHMENTS     ("attachments"),
+    DISCUSSIONS     ("discussions"),
+    CELLLINKS       ("cellLinks"),
+    FORMS           ("forms"),
+    ;
+
+    String inclusion;
+
+    SheetTemplateInclusion(String inclusion) {this.inclusion = inclusion;}
+
+    @Override
+    public String toString() { return inclusion; }
```

Comment:
Another option here, instead of using the "toString" method to get the correct value for the API call call out, you could have a common interface among the Inclusion enumes that specifies a "getApiValue" method that they'd have to implement. It helps to avoid errors in the future by requiring each enum to implement the correct method. Someone might forget the "toString" method in the future. Just a suggestion.